src/PrestaShopBundle/Form/Admin/ShopParameters/CustomerPreferences/CustomerPreferencesDataProvider.php

```diff
@@ -60,6 +69,38 @@ public function getData()
      */
     public function setData(array $data)
     {
+        if ($errors = $this->validate($data)) {
+            return $errors;
+        }
+
         return $this->generalDataConfiguration->updateConfiguration($data['general']);
     }
+
+    /**
+     * Perform validations on form data
+     *
+     * @param array $data
+     *
+     * @return array    Array of errors if any
+     */
+    private function validate(array $data)
+    {
+        $invalidFields = [];
```

Comment:
why do you need `$invalidFields`? You do a check on only one field ($data['general']['password_reset_delay'])

Reply:
not really, I added it just in case i needed more fields to validate. removed it now. :)